### PR TITLE
[Doc]: Fix code examples of List's bulk action buttons.

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: docs/List.md [hunk 1]
```diff
@@ -296,8 +296,9 @@ import {
     useNotify,
     useUnselectAll,
 } from 'react-admin';
+import { VisibilityOff } from '@material-ui/icons';
 
-const ResetViewsButton = ({ selectedIds }) => {
+const ResetViewsButton = ({ resource, selectedIds }) => {
```
Comment (fzaninotto) [inline docs/List.md]: I'd use 'posts' instead of `resource` un the `unselectAll` call. I find it more explicit.

File: docs/List.md [hunk 2]
```diff
@@ -343,7 +344,7 @@ import {
     useUnselectAll,
 } from 'react-admin';
 
-const ResetViewsButton = ({ selectedIds }) => {
+const ResetViewsButton = ({ resource, selectedIds }) => {
```
Comment (fzaninotto) [inline docs/List.md]: same

File: docs/List.md (hunk 1 above)
Comment (WiXSL) [reply docs/List.md]: Done.

File: docs/List.md (hunk 2 above)
Comment (WiXSL) [reply docs/List.md]: Done.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**